src/integrationTest/java/org/opensearch/security/http/BasicAuthWithoutChallengeTests.java

```diff
@@ -10,7 +10,7 @@
 package org.opensearch.security.http;
 
 import com.carrotsearch.randomizedtesting.annotations.ThreadLeakScope;
-import org.apache.hc.core5.http.HttpHeaders;
```

Comment:
This change should not be ported to main.

Reply:
This was intentionally - I think we should put this into main.

HttpHeaders are not specific to the the Apache Client5 - they still exist in a common namespace, so this reduces the total number of differences between the 2.x and 3.x branches.  As a future update - I think we'd want to abstract the way 'headers' are declared so only the test client file is different whereas the tests never have conflicts on imports - but that is out of scope for this change.